tests/test_npz_file.py

```diff
@@ -30,96 +30,112 @@
 
 # Populate the folder
 data = {
-    "tsd":nap.Tsd(t=np.arange(100), d=np.arange(100)),
-    "ts":nap.Ts(t=np.sort(np.random.rand(10)*100)),
-    "tsdframe":nap.TsdFrame(t=np.arange(100), d=np.random.rand(100,10)),
-    "tsgroup":nap.TsGroup({
+    "tsd": nap.Tsd(t=np.arange(100), d=np.arange(100)),
+    "ts": nap.Ts(t=np.sort(np.random.rand(10) * 100)),
+    "tsdframe": nap.TsdFrame(
+        t=np.arange(100),
+        d=np.random.rand(100, 10),
+        minfo=np.ones(10),
+    ),
+    "tsgroup": nap.TsGroup(
+        {
             0: nap.Ts(t=np.arange(0, 200)),
             1: nap.Ts(t=np.arange(0, 200, 0.5), time_units="s"),
             2: nap.Ts(t=np.arange(0, 300, 0.2), time_units="s"),
-        }),
-    "iset":nap.IntervalSet(start=np.array([0.0, 5.0]), end=np.array([1.0, 6.0]))
-    }
+        }
+    ),
+    "iset": nap.IntervalSet(start=np.array([0.0, 5.0]), end=np.array([1.0, 6.0])),
```

Comment:
I do have some testing for this in `test_metadata.py`, but since I had added the metadata test for tsdframe here, I updated it to test all three and to test versions with and without metadata (tsgroup always has metadata, so it tests with and without an extra field). I changed the tests such that metadata only gets checked in `test_load` to reduce some redundancy